services/portus/Makefile

```diff
@@ -44,7 +44,11 @@ $(if $(wildcard .env), , $(shell cp .env.config .env))
 include .env
 
 # exports
-export MACHINE_IP = $(shell hostname -I | cut -d ' ' -f1)
+# Machine host IP
+$(if $(IS_OSX), \
```

Comment:
why not calling scripts/portable.sh from here as well? try something like:
```bash
export MACHINE_IP = $(shell $(realpath $(CURDIR)/../../scripts/portable.sh) get_this_ip)
```

Reply:
done